lib/client-assets.php

```diff
@@ -114,9 +114,14 @@ function gutenberg_override_translation_file( $file, $handle ) {
 		return $file;
 	}
 
-	// Only override script handles generated from the Gutenberg plugin.
-	$packages_dependencies = include dirname( __FILE__ ) . '/packages-dependencies.php';
-	if ( ! isset( $packages_dependencies[ $handle ] ) ) {
+	// Ignore scripts whose handle does not have the "wp-" prefix.
+	if ( 'wp-' !== substr( $handle, 0, 3 ) ) {
+		return $file;
+	}
+
+	// Ignore scripts that are not found in the expected `build/` location.
+	$script_path = gutenberg_dir_path() . 'build/' . substr( $handle, 3 ) . '/index.js';
+	if ( ! file_exists( $script_path ) ) {
```

Comment:
I'm unsure how to test whether this is working well, I'd appreciate some help.

Reply:
That's a great question. Travis seems to be happy about changes applied 🤷‍♂️ 

How about we compare all script tags in the HTML output between `master` and this branch?

Reply:
This is rather tricky to verify, but it _seems_ to be working. Here's what I did:

- Build the plugin zip from this PR.
- Get a new WordPress site.
- Install the plugin from the zip.
- Change the site language.
- Go to updates and install language updates. This will download the missing language files for the Gutenberg plugin we just uploaded.
- Open the editor.
- Check a known string: `wp.i18n.__( 'Use the classic WordPress editor.' )`
- Find the _plugin_ language file containing your string. In my case `/wp-content/languages/plugins/gutenberg-es_ES-d83116db7e3459d9c9a90cae3216278f.json`
- Edit the language file changing the _translation_ of your string (`Utiliza el editor clásico de WordPress.`) for something else (`👋 Hi from the plugin language file.`)
- Activate the plugin.
- Check the string from the editor again: `wp.i18n.__( 'Use the classic WordPress editor.' )`
- Should be your new string.
- Deactivate the plugin.
- Back to the original string.